### PR TITLE
Added per channel coloring to the Well Chart

### LINKED ISSUE #672: Well Chart should color per channel
As it only consists of charts from one well they currently have always the same color which is useless for distinguishing. It also displays settings from the Plate Chart which is misleading.

### PR DESCRIPTION
**Pull request by Mailaender**
Closes https://github.com/eclipse/chemclipse/issues/672

### CODE REVIEW

File: chemclipse/plugins/org.eclipse.chemclipse.ux.extension.xxd.ui/src/org/eclipse/chemclipse/ux/extension/xxd/ui/swt/ExtendedWellChannelsUI.java [hunk 1]
```diff
@@ -91,7 +88,6 @@ private void createToolbarMain(Composite parent) {
 		//
```
Comment (eselmeister) [inline chemclipse/plugins/org.eclipse.chemclipse.ux.extension.xxd.ui/src/org/eclipse/chemclipse/ux/extension/xxd/ui/swt/ExtendedWellChannelsUI.java:88]: When changing an element, the size needs to be adjusted if elements are added/removed:
`composite.setLayout(new GridLayout(2, false)); `

File: chemclipse/plugins/org.eclipse.chemclipse.ux.extension.xxd.ui/src/org/eclipse/chemclipse/ux/extension/xxd/ui/swt/ExtendedWellDataUI.java [hunk 2]
```diff
@@ -121,23 +120,6 @@ private void createToolbarMain(Composite parent) {
 		buttonToolbarEdit = createButtonToggleToolbar(composite, toolbarEdit, IMAGE_EDIT, TOOLTIP_EDIT);
 		buttonTableEdit = createButtonToggleEditTable(composite, tableViewer, IMAGE_EDIT_ENTRY);
 		buttonDelete = createButtonDelete(composite);
-		createSettingsButton(composite);
```
Comment (eselmeister) [inline chemclipse/plugins/org.eclipse.chemclipse.ux.extension.xxd.ui/src/org/eclipse/chemclipse/ux/extension/xxd/ui/swt/ExtendedWellDataUI.java:124]: When changing an element, the size needs to be adjusted if elements are added/removed:
`composite.setLayout(new GridLayout(5, false)); `

File: chemclipse/plugins/org.eclipse.chemclipse.ux.extension.xxd.ui/src/org/eclipse/chemclipse/ux/extension/xxd/ui/swt/editors/ExtendedPCRPlateUI.java [hunk 3]
```diff
@@ -75,7 +72,6 @@ private void createToolbarMain(Composite parent) {
 		comboSubsets = createComboSubsets(composite);
```
Comment (eselmeister) [inline chemclipse/plugins/org.eclipse.chemclipse.ux.extension.xxd.ui/src/org/eclipse/chemclipse/ux/extension/xxd/ui/swt/editors/ExtendedPCRPlateUI.java:72]: When changing an element, the size needs to be adjusted if elements are added/removed:
`composite.setLayout(new GridLayout(4, false)); `

**Comment by Mailaender** [comment]
Well spotted. Updated.

**Pull request merged**